packages/list/ListItem.tsx

```diff
@@ -54,15 +54,54 @@ export default class ListItem<T extends HTMLElement = HTMLElement> extends React
     onBlur: () => {},
     onDestroy: () => {},
     tag: 'li',
+    handleClick: () => {},
```

Comment:
The naming here seems inconsistent. You're using `onBlur` and `handleClick`. I recommend being consistent with your event handler names (and generally prefer `on{EventName}`)

Reply:
`onClick` is a native React click handler. I don't want to collide and override this behavior since its native to React. Therefore I must use a different name (handleClick) so that I don't collide with the naming.

Reply:
:+1: